FreeRTOS-Plus/Source/Application-Protocols/network_transport/sockets_wrapper/cellular/sockets_wrapper.c

```diff
@@ -285,7 +285,7 @@ static BaseType_t prvNetworkRecvCellular( const cellularSocketWrapper_t * pCellu
 
     ( void ) xEventGroupClearBits( pCellularSocketContext->socketEventGroupHandle,
                                    SOCKET_DATA_RECEIVED_CALLBACK_BIT );
-    socketStatus = Cellular_SocketRecv( CellularHandle, cellularSocketHandle, buf, len, &recvLength );
```

Comment:
The `RYZ014A` cellular modem will return an error on the socket receive AT command if there is no data available. It expects that the socket receive URC will be checked first. This could be the case with other cellular modems as well. The change here removes checking of the first `Cellular_SocketRecv` call so that this doesn't fail if there is no data available. It can still time out for the second `Cellular_SocketRecv` or return an error there.

Reply:
@chinglee-iot I will look into returning 0 when data is received with the RYZ in order to match the other implementations.

Reply:
I found a way around the behavior for the RYZ and reverted this change.